modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/workspaces/AddWorkspaceForm.tsx

```diff
@@ -55,10 +58,8 @@ const getProjectLocations = (): {label: string; value: string}[] => {
 
 const getDefaultServerLocation = (): string => {
 	switch (FARO_ENV) {
-		case 'dev':
-			return DEV;
-		case 'uat':
-			return UAT;
+		case FaroEnv.Staging:
+			return INTERNAL;
```

Comment:
@interaminense `INTERNAL` is not the staging env.

Reply:
Which server location should appear to the env that is using stating?